docs/source/models/intervention_models/wasting_treatment/compas_trial_wasting_treatment/index.rst

```diff
@@ -294,10 +294,11 @@ Currently, we will be using the first two of these (WAZ and/or age) for targetin
 This accounts for 57% of all MAM cases. In the future, we plan to also include the third criterion 
 around MUAC but have not completed this currently. 
```

Comment:
Remove reference to third category since this is included now below

Reply:
Thanks for catching that - just updated it.